coalib/output/ConsoleInteraction.py

```diff
@@ -929,7 +929,8 @@ def print_bears(bears,
     :param console_printer:  Object to print messages on the console.
     :param args:             Args passed to coala command.
     """
-    if not bears:
+    json_flag = args and args.json
+    if not bears and not json_flag:
```

Comment:
up to you: maybe inlining `args and args.json`

Reply:
Done :+1:

Reply:
Thanks for your suggestion :) It made the code more clear and removed the unnecessary use of a variable.